src/component/Cell.tsx

```diff
@@ -5,6 +5,6 @@ import type { FunctionComponent, SVGProps } from 'react';
 
 export type Props = SVGProps<SVGElement>;
 
-export const Cell: FunctionComponent<Props> = (props: Props) => null;
+export const Cell: FunctionComponent<Props> = (_props: Props) => null;
```

Comment:
Do the components use `_props` used at all?

Reply:
yes, as far as I know
https://recharts.org/en-US/api/Cell